frontend/src/components/PostRecipe.jsx

```diff
@@ -1,308 +1,598 @@
+            </Col>
+            <Col xs={5}>
+              <h2 id="title">Post recipe</h2>
+            </Col>
+            <Col xs={2}></Col>
+          </Row>
+          <Row>
+            <Col sm={3} md={3} lg={3}>
+              <CSSTransition
+                in={true}
+                timeout={500}
                 classNames="slideUp"
                 appear
-            >
-                <div className="recipe-container-post-recipe">
-                    <div className="recipe-form">
-                        <h2 id='title'>Post recipe</h2>
-    
-                        <div className="input-section">
-                            <label id='subtitle'>Recipe Name</label>
-                            <input id='input_postRecipe' 
-                                type="text" 
-                                value={recipeName} 
-                                onChange={(e) => setRecipeName(e.target.value)} 
-                                placeholder="Enter Recipe Name" 
-                            />
-                        </div>
-                        
-                        <div className="input-subsection">
-                            <div className="ingredients-section">
-                                <label>INGREDIENTS</label>
-                                <div className="ingredient-list">
-                                    {ingredients.map((ingredient, index) => (
-                                        <div key={index} className="ingredient-item">
-                                            <span>{ingredient.name} - {ingredient.quantity} {ingredient.unit}</span>
-                                            <button id='buttons_postRecipe'  onClick={() => handleIngredientDelete(index)}>X</button>
-                                        </div>
-                                    ))}
-                                </div>
-                                <div className="ingredient-input-group">
-                                    <input id='input_postRecipe-ingredient' 
-                                        type="text" 
-                                        ref={ingredientNameRef}
-                                        placeholder="Ingredient Name"
-                                    />
-    
-                                    <input id='input_postRecipe' 
-                                        type="number" 
-                                        ref={ingredientQuantityRef}
-                                        placeholder="Quantity"
-                                    />
-                                    <select className="ingredient-unit-select"
-                                        ref={ingredientUnitRef}
+              >
+                <Container
+                  id="recipe-container"
+                  className="mt-3 rounded box-shadow"
+                >
+                  <Row>
+                    <Col xs={12}>
+                      <div className="ingredient-list">
```

Comment:
Redundant dive, you can move the classname to the column.

Reason: Col is a div, with a specific class, having an inner just makes more identation and excessive nesting.